docs/pages/appbars/index.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+
+  <DocsPageTemplate>
+
+    <DocsPageSection title="Overview" anchor="#overview">
+      <p>
+        App bars are used to provide users a means of navigation through pages in our applications. They allow access to top-level pages, navigate back to a specific page, or dismiss edit forms
+      </p>
+
+      <p>
+        We have four types of app bars: <em>primary bars</em>, <em>back arrow bars</em>, <em>form bars</em>, and <em>bottom bars</em>
+      </p>
+
+      <h3>Primary bars</h3>
+      <p>Primary bars are placed on top level pages in Kolibri. They may house subtabs that lead to different sections of the application</p>
+      <DocsShow>
+        <img class="modal-img" src="./primary-1.png">
+      </DocsShow>
+      <DocsShow>
+        <img class="modal-img" src="./primary-2.png">
+      </DocsShow>
+
+      <h3>Back arrow bars</h3>
+      <p>Back arrow topbars allow users to navigate back to the previous page</p>
+      <DocsShow>
+        <img class="modal-img" src="./back-1.png">
+      </DocsShow>
+
+      <h3>Form bars</h3>
+      <p>Form bars allows users to dismiss a full-screen form and land on the previous page</p>
+      <DocsShow>
+        <img class="modal-img" src="./form-1.png">
+      </DocsShow>
+
+      <h3>Bottom bars</h3>
+      <p>Bottom bars are placed on pages that house editing, content selection, or multi-step processes. They may contain buttons that allow the user to submit specific action(s) or continue with the process. Bottom bars can also contain contextual information and error messaging relevant to the task</p>
```

Comment:
I think it'd be helpful to specify when we should and shouldn't use a bottom bar, and also what type of metadata is and isn't allowed here. I remember you mentioned there's specific kind of metadata that would go on the left side of the bottom bar vs the metadata that would appear right next to the submit/cancel buttons

Reply:
We decided that further bottom bar specifications would be housed in the single task page pattern doc (on the way). This is why I generalized it here as "contextual information"

Reply:
I can a few layout pointers in the layout section. But as for "when to use", they should only be used on single task pages